Sources/Core/GTMSessionFetcher.m

```diff
@@ -3995,6 +4019,21 @@ - (void)setUsingBackgroundSession:(BOOL)flag {
   }  // @synchronized(self)
 }
 
+- (BOOL)stopFetchingTriggersCompletionHandler {
+  return _stopFetchingTriggersCompletionHandler;
+}
+
+- (void)setStopFetchingTriggersCompletionHandler:(BOOL)flag {
+  @synchronized(self) {
+    GTMSessionMonitorSynchronized(self);
+    GTMSESSION_ASSERT_DEBUG(
+        _initialBeginFetchDate == nil,
+        @"stopFetchingTriggersCompletionHandler should not change after fetcher starts");
+
+    _stopFetchingTriggersCompletionHandler = flag;
```

Comment:
This should probably still have a guard so it won't update if _initialBeginFetchDate is non nil.

Reply:
Something like:

```
  if (_initialBeginFetchDate == nil) {
    _stopFetchingTriggersCompletionHandler = flag;
  } else {
    GTMSESSION_ASSERT_DEBUG(
        0,
        @"stopFetchingTriggersCompletionHandler should not change after fetcher starts");
  }
```

?

Reply:
Updated